### PR TITLE
Deborah/idv document rejected

### PR DESCRIPTION
**Pull request by deborah-deriv**
### Deborah/idv document rejected
This is the PR for IDV document rejected

### CODE REVIEW

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    cy.get('.dc-checkbox__box').click()
+    cy.findByRole('button', { name: 'Verify' }).should('be.enabled').click()
+    cy.contains("Your documents were submitted successfully").should('be.visible')
+    cy.findByText("Proof of address required").should('exist')
+    cy.reload()
+
+    cy.contains("Your identity verification failed").should('be.visible')
+    cy.findByText("Proof of address required").should('exist')
+    cy.c_closeNotificationHeader()
+    cy.contains("We were unable to verify the identity document with the details provided.").should('be.visible')
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/KYC/idvDocumentRejected.cy.js:26]: use this instead
```suggestion
    cy.findByText("We were unable to verify the identity document with the details provided.").should('be.visible')
```

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    cy.get('select[name="document_type"]').select('Passport')
+    cy.findByLabelText('Enter your document number').type('G0000001')
+    cy.findByTestId('first_name').clear().type('Joe Doe')
+    cy.findByTestId('last_name').clear().type('Leo')
+    cy.findByTestId('date_of_birth').type('2000-09-20')
+
+    cy.findByRole('button', { name: 'Verify' }).should('be.disabled')
+    cy.get('.dc-checkbox__box').click()
+    cy.findByRole('button', { name: 'Verify' }).should('be.enabled').click()
+    cy.contains("Your documents were submitted successfully").should('be.visible')
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/KYC/idvDocumentRejected.cy.js:19]: use this instead
```suggestion
    cy.findByText("Your documents were submitted successfully").should('be.visible')
```

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    cy.findByText("Proof of address required").should('exist')
+    cy.reload()
+
+    cy.contains("Your identity verification failed").should('be.visible')
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/KYC/idvDocumentRejected.cy.js:23]: use this instead
```suggestion
    cy.findByText("Your identity verification failed").should('be.visible')
```

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    cy.findByText("Proof of address required").should('exist')
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/KYC/idvDocumentRejected.cy.js:24]: assert on visibility instead
```suggestion
    cy.findByText("Proof of address required").should('be.visible')
```

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    cy.findByLabelText('Enter your document number').type('G0000001')
+    cy.findByTestId('first_name').clear().type('Joe Doe')
+    cy.findByTestId('last_name').clear().type('Leo')
+    cy.findByTestId('date_of_birth').type('2000-09-20')
+
+    cy.findByRole('button', { name: 'Verify' }).should('be.disabled')
+    cy.get('.dc-checkbox__box').click()
+    cy.findByRole('button', { name: 'Verify' }).should('be.enabled').click()
+    cy.contains("Your documents were submitted successfully").should('be.visible')
+    cy.findByText("Proof of address required").should('exist')
```
Comment (bilalmajid-deriv) [inline cypress/e2e/wip/KYC/idvDocumentRejected.cy.js:20]: assert on visibility instead
```suggestion
    cy.findByText("Proof of address required").should('be.visible')
```

**Review by bilalmajid-deriv — CHANGES_REQUESTED** [review]
Changes requested

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js (hunk 1 above)
Comment (deborah-deriv) [reply cypress/e2e/wip/KYC/idvDocumentRejected.cy.js]: This is addressed

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js (hunk 3 above)
Comment (deborah-deriv) [reply cypress/e2e/wip/KYC/idvDocumentRejected.cy.js]: I have removed this line as this notification is not showing

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js (hunk 2 above)
Comment (deborah-deriv) [reply cypress/e2e/wip/KYC/idvDocumentRejected.cy.js]: This is addressed

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js (hunk 4 above)
Comment (deborah-deriv) [reply cypress/e2e/wip/KYC/idvDocumentRejected.cy.js]: This is addressed

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js (hunk 5 above)
Comment (deborah-deriv) [reply cypress/e2e/wip/KYC/idvDocumentRejected.cy.js]: This is addressed

**Review by bilalmajid-deriv — APPROVED** [review]
LGTM

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    cy.findByLabelText('Enter your document number').type('G0000001')
+    cy.findByTestId('first_name').clear().type('Joe Doe')
+    cy.findByTestId('last_name').clear().type('Leo')
+    cy.findByTestId('date_of_birth').type('2000-09-20')
+
+    cy.findByRole('button', { name: 'Verify' }).should('be.disabled')
+    cy.get('.dc-checkbox__box').click()
+    cy.findByRole('button', { name: 'Verify' }).should('be.enabled').click()
+    cy.findByText("Your documents were submitted successfully").should('be.visible')
+    cy.findByText("Proof of address required").should('be.visible')
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/KYC/idvDocumentRejected.cy.js:20]: Can you please change these double quotes to single for consistency's sake?

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+    cy.reload()
+
+    cy.findByText("Proof of address required").should('be.visible')
+    cy.c_closeNotificationHeader()
+    cy.findByText("We were unable to verify the identity document with the details provided.").should('be.visible')
```
Comment (abuzar-deriv) [inline cypress/e2e/wip/KYC/idvDocumentRejected.cy.js:25]: Same as above for this.

**Review by abuzar-deriv — CHANGES_REQUESTED** [review]

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js (hunk 6 above)
Comment (deborah-deriv) [reply cypress/e2e/wip/KYC/idvDocumentRejected.cy.js]: This is addressed

File: cypress/e2e/wip/KYC/idvDocumentRejected.cy.js (hunk 7 above)
Comment (deborah-deriv) [reply cypress/e2e/wip/KYC/idvDocumentRejected.cy.js]: this is addressed

**Review by abuzar-deriv — APPROVED** [review]
Looks good.

**Review by meeyun-deriv — APPROVED** [review]
LGTM

**Pull request merged**